pkgs/by-name/ca/capnproto/package.nix

```diff
@@ -40,12 +43,16 @@ clangStdenv.mkDerivation rec {
     hash = "sha256-aDcn4bLZGq8915/NPPQsN5Jv8FRWd8cAspkG3078psc=";
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
   propagatedBuildInputs = [
     openssl
     zlib
   ]
-  ++ lib.optional (clangStdenv.cc.isClang && clangStdenv.hostPlatform.isStatic) empty-libgcc_eh;
+  ++ lib.optional (clangStdenv.cc.isClang && clangStdenv.hostPlatform.isStatic) empty-libgcc_eh
+  ++ lib.optional clangStdenv.hostPlatform.isMusl libucontext;
```

Comment:
Should this be `!clangStdenv.hostPlatform.isGnu`? What's right for non-Linux systems?

Reply:
Well, Apple isn't the only other platform, but it does look like basically everything that isn't musl has ucontext.h, so I guess this is fine.